assets/airports/ktpa.json

```diff
@@ -0,0 +1,1566 @@
+{
+    "radio": {
+        "twr": "Tampa Tower",
+        "app": "Tampa Approach",
+        "dep": "Tampa Departure"
+    },
+    "icao": "KTPA",
+    "iata": "TPA",
+    "magnetic_north": -5.3,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 6000,
+    "position": ["N27d58m31.8", "W82d31m59.6", "26ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N27d58m31.8", "W82d31m59.6"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 220,
+        "speed": 5
+    },
+    "arrivalRunway": "19L",
+    "departureRunway": "19R",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 12000,
+            "airspace_class": "B",
+            "poly": [
+                ["N28.74164833", "W82.71669361"],
+                ["N28.70662639", "W82.49635250"],
+                ["N28.68570361", "W82.36523611"],
+                ["N28.65170889", "W82.15468333"],
+                ["N28.64517194", "W82.11489500"],
+                ["N28.63327917", "W82.04166806"],
+                ["N28.56389417", "W81.95692333"],
+                ["N28.32894028", "W81.95530083"],
+                ["N28.30772972", "W81.94919944"],
+                ["N28.19533750", "W81.91752194"],
+                ["N28.10020000", "W81.89076944"],
+                ["N28.00846972", "W81.83190500"],
+                ["N27.92794389", "W81.83171194"],
+                ["N27.78299417", "W81.98085139"],
+                ["N27.76680833", "W82.06589528"],
+                ["N27.68102250", "W82.11139778"],
+                ["N27.65511000", "W82.12509778"],
+                ["N27.58450639", "W82.16242639"],
+                ["N27.41477833", "W82.25202361"],
+                ["N27.39196472", "W82.26297611"],
+                ["N27.24553278", "W82.33415833"],
+                ["N27.18329194", "W82.73331722"],
+                ["N27.49996167", "W83.10835361"],
+                ["N27.53528417", "W83.14866250"],
+                ["N28.15830194", "W83.20832944"],
+                ["N28.27375250", "W83.10793389"],
+                ["N28.30531639", "W83.08094972"],
+                ["N28.55532333", "W82.86782472"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_1L187PIE116": ["N27.848198", "W82.546200"],
+        "_1R187PIE116": ["N27.842546", "W82.533100"],
```

Comment:
`_RWY1LPIE116` would probably be more consistent with the other forms we've used.

Reply:
done 👍